lms/djangoapps/course_blocks/management/commands/generate_course_blocks.py

```diff
@@ -29,85 +35,130 @@ def add_arguments(self, parser):
         Entry point for subclassed commands to add custom arguments.
         """
         parser.add_argument(
-            '--all',
-            help='Generate course blocks for all or specified courses.',
-            action='store_true',
-            default=False,
+            '--courses',
+            dest='courses',
+            nargs='+',
+            help='Generate course blocks for the list of courses provided.',
         )
         parser.add_argument(
-            '--dags',
-            help='Find and log DAGs for all or specified courses.',
+            '--all_courses',
+            help='Generate course blocks for all courses, given the requested start and end indices.',
             action='store_true',
             default=False,
         )
         parser.add_argument(
-            '--force',
-            help='Force update of the course blocks for the requested courses.',
+            '--enqueue_task',
+            help='Enqueue the tasks for asynchronous computation.',
             action='store_true',
             default=False,
         )
         parser.add_argument(
-            '--verbose',
-            help='Enable verbose logging.',
+            '--routing_key',
+            dest='routing_key',
+            help='Routing key to use for asynchronous computation.',
+        )
+        parser.add_argument(
+            '--force_update',
+            help='Force update of the course blocks for the requested courses.',
             action='store_true',
             default=False,
         )
         parser.add_argument(
-            '--start',
-            help='Starting index of course.',
+            '--start_index',
+            help='Starting index of course list.',
             default=0,
             type=int,
         )
         parser.add_argument(
-            '--end',
-            help='Ending index of course.',
+            '--end_index',
+            help='Ending index of course list.',
             default=0,
             type=int,
         )
+        parser.add_argument(
+            '--dags',
+            help='Find and log DAGs for all or specified courses.',
```

Comment:
Valid point.  It's true that this usage of the term DAG is strictly incorrect.  Historically, this came from us asking the question "should edX support DAGs (as opposed to simple Trees) in the courseware".   For now, I'll rename its usage in this file to multi-parent.